kubemarine/plugins/__init__.py

```diff
@@ -439,7 +439,7 @@ def expect_pods(cluster, pods, namespace=None, timeout=None, retries=None,
             cluster.log.debug(running_pods_stdout)
             time.sleep(timeout)
 
-    raise Exception('In the expected time, the pods did not become ready')
+    raise Exception('In the expected time, the pods did not become ready. Try to increase number of retries in expect.pods: https://github.com/Netcracker/KubeMarine/blob/main/documentation/Installation.md#expect-pods')
```

Comment:
This will be misleading if the method is invoked from `kubernetes.wait_for_any_pods`. In that case the timeout is not configurable.

Reply:
Removed for now. Maybe it's worth to consider making both timeouts `pods.expect.plugins` and `pods.expect.kubernetes` configurable via `globals:{}`.